<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewDiplomaDialog</class>
 <widget class="QDialog" name="AddNewDiplomaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Diploma Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Institut</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Schwer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDiploma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSprache">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboInstitut">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSchwer">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSave">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
